src/lib/viewers/controls/color-picker/ColorPickerControl.tsx

```diff
@@ -44,7 +44,13 @@ export default function ColorPickerControl({
                 type="button"
                 {...rest}
             >
-                <div className="bp-ColorPickerControl-swatch" style={{ backgroundColor: activeColor }} />
+                <div
+                    className={classNames('bp-ColorPickerControl-highlight', {
```

Comment:
Can we avoid the word "highlight" here, since it has an existing meaning for annotations (text highlights)? How about `bp-ColorPickerControl-toggle-background`?